src/util/validations.ts

```diff
@@ -83,21 +83,22 @@ export const manifestSchema = z.object({
  * Validates given `manifest` object to match pattern.
  */
 export const validateManifest = (manifest: any) =>
-  validate(manifestSchema, manifest, ManifestValidationError);
+  validate(manifestSchema, manifest, undefined, ManifestValidationError);
```

Comment:
why undefined?

Reply:
The `validate` function is used by the `validateManifest` function, which is called from plugins too. To avoid creating a new function, I assigned `index` to `undefined` here.

Reply:
in that case either make it last argument, or pass all the info to validate function as a single object

Reply:
Both options are not achievable because a plugin calls this version of the validate function: validate<z.infer<typeof schema>>(schema, input, index). If we change it to a single object, all plugins will need to be updated accordingly. If we change the order of the arguments, we would need to provide an Error class in every plugin. I hope I have described the situation :)